Gemfile

```diff
@@ -12,7 +12,12 @@ end
 
 gemspec
 
-gem 'rake'
+if RUBY_VERSION < '1.9.3'
```

Comment:
instead of this, could we drop ruby1.8 support? This is so old and EOL since a long time. voxpupuli started to drop ruby1.8 for each new module release a few months back.